.eslintrc.json

```diff
@@ -19,7 +19,9 @@
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
+    "no-return-await":0,
     "comma-dangle": 0,
+    "import/no-cycle": 0,
```

Comment:
Hi @King-Success . Are there any reasons for turning off the `no-return-await` rule and the `import/no-cycle`?. I am asking because the configuration in this file is a strict requirement for the project.

Reply:
Oops my bad, I was experimenting and forgot to reverse afterward, thanks for pointing that out.